typings/index.d.ts

```diff
@@ -1183,12 +1182,7 @@ declare module 'discord.js' {
   export class InteractionWebhook extends PartialWebhookMixin() {
     constructor(client: Client, id: Snowflake, token: string);
     public token: string;
-    public send(
-      options: string | APIMessage | (InteractionReplyOptions & { split?: false }),
-    ): Promise<Message | RawMessage>;
-    public send(
-      options: APIMessage | (InteractionReplyOptions & { split: true | SplitOptions }),
-    ): Promise<(Message | RawMessage)[]>;
+    public send(options: string | APIMessage | InteractionReplyOptions)): Promise<Message | RawMessage>;
```

Comment:
Isn't there a bracket more?
```suggestion
    public send(options: string | APIMessage | InteractionReplyOptions): Promise<Message | RawMessage>;
```